NEWS.md

```diff
@@ -1,5 +1,6 @@
 ## 20.3.0 IN-PROGRESS
 
+* Added code to close database connections when encountering errors from the /inventory-hierarchy endpoint (MODINVSTOR-677)
```

Comment:
I prefer that release notes describe the way the change impacts the clients / users of the module

```suggestion
* Closes database connections when encountering errors in the /inventory-hierarchy endpoint (MODINVSTOR-677)
```